### ISSUE TITLE
[Accessibility] Cannot move focus between articles via arrow keys

### ISSUE THREAD

**Issue by jooyoungseo** [zotero/zotero#4144]
I know users are recommended to use Zotero Forum instead of GitHub issue; however, please understand that GitHub interface is much more accessible to screen reader users to interact with and share the bugs and I hope you don't prevent users from sharing their voice through their preferred channels.

I have been using Zotero 7 Beta on Windows 11 with JAWS and NVDA. Definitely, I have been able to experience its improved accessibility. I appreciate it!

However, one of the essential keyboard features that used to work does not seem to work in beta. I cannot move and select articles in the list view via up/down arrow keys. I was able to move my focus by using ctrl+DownArrow, but this does not actually select the item. Please refer to the gif below:

![Recording 2024-05-21 at 08 22 53](https://github.com/zotero/zotero/assets/19754711/9f1b32f6-b7dd-46d1-9015-d4a61fba82f9)

This is a critical issue to screen reader and keyboard-only users as we cannot easily choose the article that we want.

**Comment by abaevbog** [comment]
@jooyoungseo I am not yet able to reproduce this though there is a similar report to this: #4030

To be clear, you are saying that pressing `arrowDown` from the top-most item "A generalized transition network ..." did not select the item below it ("A layered Grammar of Graphics")? Did it do anything or just nothing? The selected article does change later in the gif - what are you pressing for it to get selected?

I assume this happens with a screen reader active? Which one?

Finally, this happens with all plugins disabled, right? I doubt a plugin would cause this but just to make sure.

This is certainly an issue, so it's just a matter of reproducing this - feel free to include any additional details that would help me run into this behavior

**Comment by jooyoungseo** [comment]
@abaevbog Okay, I have figured this out. NVDA and Narrator do not have this issue. It has to do with JAWS. With JAWS, what's visually focused is not matched with the speech announcement. When JAWS users DownArrow between the items, the speech synthesizer reads aloud the tag edit field, such as "abstract," "title," "first," etc.

After you test this with JAWS, please check the JAWS speech history by pressing JAWS+Space Bar followed by H key.

**Comment by jooyoungseo** [comment]
@abaevbog This is new recording and I have copied the JAWS speech history for your reference.

## gif

![Recording 2024-05-21 at 13 29 21](https://github.com/zotero/zotero/assets/19754711/71ebbebd-4c00-43ba-a3f5-d3d73bd310a4)


## JAWS Speech

```md
a11y_viz - Zotero
A generalized transition network representation for interactive systems. Conference Paper. Kieras and Polson Has PDF attachment. Has PDF attachment closed
1 of 29
Abstract edit
Abstract edit
Abstract edit
Abstract edit
Edit
practicum education
Abstract edit
Abstract edit
Abstract edit
Abstract edit
(first) Edit
Arvind
Edit
CCS Concepts
(first) Edit
Y.
(first) Edit
Edit
STEM Accessibility
```

**Comment by abaevbog** [comment]
@jooyoungseo one of the JAWS issues should be fixed in the latest build. On my end, now JAWS not does not get confused by the side pane while navigating the items list with arrows and item names are properly announced. Could you confirm if this issues is resolved for you? Make sure you are on the latest beta build (Help > Check for updates).

**Comment by jooyoungseo** [comment]
Yes! It works nicely with JAWS now! Thanks for addressing this issue.

**Comment by abaevbog** [comment]
Fantastic!
